2dcontext/drawing-images-to-the-canvas/2d.drawImage.incomplete.emptysrc.html

```diff
@@ -19,15 +19,18 @@
 var t = async_test("");
 _addTest(function(canvas, ctx) {
 
-ctx.fillStyle = '#0f0';
+ctx.fillStyle = '#f00';
 ctx.fillRect(0, 0, 100, 50);
-var img = document.getElementById('red.png');
+var img = document.getElementById('green.png');
 img.src = "";
```

Comment:
It seems this case is sync in Chromium and Gecko at least...
Filed https://github.com/whatwg/html/issues/2429